content/html/concepts/elements/terms/del/del.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: '<del>'
+Description: 'Shows text that is to be removed from a document, usually rendered in strike-through text.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+  Tags:
+  - 'Elements'
+  - 'Tags'
+  - 'Style'
+  CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
```

Comment:
```suggestion
Tags:
  - 'Elements'
  - 'Tags'
  - 'Style'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
```

Just the list items for `Subjects`, `Tags`, and `CatalogContent` are indented by 2 spaces. 😄

Reply:
done